soliket/gaussian/gaussian_data.py

```diff
@@ -163,5 +170,6 @@ def plot_cov(self, **kwargs):
             for j, lj in zip(range(len(self.data)), self.labels)
         ]
 
-        return hv.HeatMap(data).opts(tools=["hover"], width=800, height=800,
-                                     invert_yaxis=True, xrotation=90)
+        return hv.HeatMap(data).opts(
+            tools=["hover"], width=800, height=800, invert_yaxis=True, xrotation=90
+        )
```

Comment:
Is this whole function necessary? I would be in favour of removing it and letting people plot their own covmats. At the moment it adds an extra, fairly well hidden, dependency.